overlord/fdestate/backend/reseal.go

```diff
@@ -119,76 +119,85 @@ func getUniqueModels(bootChains []boot.BootChain) []secboot.ModelForSealing {
 	return models
 }
 
+type resealParamsAndLocation struct {
+	params   *SealingParameters
+	location secboot.KeyDataLocation
+}
+
 func doReseal(manager FDEStateManager, method device.SealingMethod, rootdir string) error {
-	runParams, err := manager.Get("run+recover", "all")
+	runParamsData, err := manager.Get("run+recover", "system-data")
```

Comment:
this will fallback with to containerRole all anyway on the transition?

Reply:
Yes. `FdeState.getParameters`, takes the parameters from `"all"` if specific parameters for that role.

Reply:
> Yes. `FdeState.getParameters`, takes the parameters from `"all"` if specific parameters for that role.

did you mean *if it has no specific parameters..*?